### PR TITLE
🌱 Release v1.6.0-rc.1

### PR DESCRIPTION
**Pull request by g-gaston**
/area release

### CODE REVIEW

File: CHANGELOG/v1.6.0-rc.1.md [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+* Types in `ipam.cluster.x-k8s.io` have graduated to `v1beta1`
+
+## Deprecation Warning
+
+- The API version `v1alpha4` is no longer served in this release.
+  - [Version migration guide](https://main.cluster-api.sigs.k8s.io/developer/providers/version-migration.html)
+  - [GitHub issue #8038](https://github.com/kubernetes-sigs/cluster-api/issues/8038)
+  - [API deprecation details](https://main.cluster-api.sigs.k8s.io/contributing#removal-of-v1alpha3--v1alpha4-apiversions)
+- The API version `v1alpha3` has been completely removed in this release.
+- Flag `--metrics-bind-addr` is [deprecated](https://github.com/kubernetes-sigs/cluster-api/pull/9264) for all controlle)
```
Comment (cahillsf) [inline CHANGELOG/v1.6.0-rc.1.md:16]: > for all controlle)

looks like a typo -- "for all controllers"?

File: CHANGELOG/v1.6.0-rc.1.md [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+- Runtime SDK: Set User Agent for test extension correctly (#9748)
+- util: Fix AddAnnotations for unstructured.Unstructured (#9164)
+
+## :seedling: Others
+- API: Add ClusterClass column to Cluster CRD (#9120)
+- API: Add verify-import-restrictions to enforce import restrictions (#9407)
+- API: Enforce import restrictions in all API packages (#9461)
+- API: Ensure we generate all webhook manifests.yaml (#9621)
+- API: Move API v1beta1 webhooks to a separate package (#9047)
+- API: Move docker  infrastructure experimental API v1beta1 webhooks to sepa… (#9460)
```
Comment (furkatgofurov7) [inline CHANGELOG/v1.6.0-rc.1.md:114]:
```suggestion
- API: Move docker infrastructure experimental API v1beta1 webhooks to separate package (#9460)
```

**Review by furkatgofurov7 — COMMENTED** [review]
Thank you for PR.

Few nits + we nee to remove redundant dependancy bumps. otherwise LGTM

File: CHANGELOG/v1.6.0-rc.1.md [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+- ClusterClass: Add topology-owned label to MachineHealthChecks. (#9191)
+- ClusterClass: Introduce NamingStrategy and allow generating names using go templates (#9340)
+- ClusterClass: Update API with ClusterClass MachinePool support (#8820)
+- clusterctl: Add RKE2 bootstrap provider to clusterctl (#9720)
+- clusterctl: Block move with annotation (#8690)
+- IPAM: Promote IPAM types to v1beta1 (#9525)
+- MachinePool: Add MachinePool workers support in ClusterClass (#9016)
+- MachineSet: Adjust preflight check to allow kubelet version skew of 3 for clusters running v1.28 and above (#9222)
+- Testing/Documentation: V1.28: Prepare quickstart, capd and tests for the new release including kind bump (#9160)
+- Release: Add automation to create release branch and tags (#9111)
```
[the remark this answers is not in the document]
Comment (g-gaston) [reply CHANGELOG/v1.6.0-rc.1.md:70]: you are right, taking a note to check the release tool

**Review by furkatgofurov7 — APPROVED** [review]
Thank you!

/approve

**Review by willie-yao — APPROVED** [review]
/lgtm

**Pull request merged**